components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -89,6 +103,63 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem {
   }
 
   _selectedItem = selectedItem;
+  if (itemIndex == NSNotFound) {
+    return;
+  }
+
+  UIView *itemView = self.itemViews[itemIndex];
+  if ([itemView isKindOfClass:[MDCTabBarViewItemView class]]) {
+    MDCTabBarViewItemView *selectedItemView = (MDCTabBarViewItemView *)itemView;
+    selectedItemView.iconImageView.image = selectedItem.image;
+  }
+}
+
+#pragma mark - Key-Value Observing (KVO)
+
+- (void)addObserversToTabBarItems {
+  for (UITabBarItem *item in self.items) {
```

Comment:
What's special about image and title that we only listen to these?

Reply:
They're the only two being added in this PR. We can add other properties later (like those for UIAccessibility or titleTextAttributes) once we have more of the component built.